data_diff/dbt.py

```diff
@@ -105,7 +109,7 @@ def dbt_diff(
                 + " <> "
                 + ".".join(diff_vars.dev_path)
                 + "[/] \n"
-                + "Skipped due to missing primary-key tag(s).\n"
+                + "Skipped due to unknown primary key. Add uniqueness tests, meta, or tags.\n"
```

Comment:
could probably use some wordsmithing

Reply:
I think you're explaining the error and giving actionable feedback, so the message seems ok to me.

Couple ideas if you're looking for options or ideas

> Skipped due to unknown primary key. Model primary keys must have a uniqueness test, or  have a "primary-key" value in the meta or tag configuration.

> Skipped due to unknown primary key. Help data-diff infer the primary key by adding uniqueness tests, meta, or tags.

Reply:
I think I will leave it. Always a trade off between brevity/info